### PR TITLE
add status and secret handling to controller

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: operator/controllers/lemming_controller.go [hunk 1]
```diff
@@ -69,10 +84,83 @@ func (r *LemmingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
...
+	default:
+		lemming.Status.Phase = lemmingv1alpha1.Unknown
+	}
+	lemming.Status.Message = fmt.Sprintf("Pod Details: %s", pod.Status.Message)
+	if err := r.Status().Update(ctx, lemming); err != nil {
+		log.Error(err, "unable to update lemming status")
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
```
Comment (wenovus) [inline operator/controllers/lemming_controller.go:101]: Is the empty result intentional?

File: operator/controllers/lemming_controller.go [hunk 2]
```diff
@@ -96,19 +184,42 @@ func (r *LemmingReconciler) reconcilePod(ctx context.Context, lemming *lemmingv1
 	pod.Spec.Containers[0].Env = lemming.Spec.Env
 	pod.Spec.Containers[0].Resources = lemming.Spec.Resources
 
+	var hasVolume bool
```
Comment (wenovus) [inline operator/controllers/lemming_controller.go:187]: hasTLSVolume?

**Review by wenovus — APPROVED** [review]

File: operator/controllers/lemming_controller.go (hunk 2 above)
Comment (DanG100) [reply operator/controllers/lemming_controller.go]: done

File: operator/controllers/lemming_controller.go (hunk 1 above)
Comment (DanG100) [reply operator/controllers/lemming_controller.go:101]: yeah, non-empty allows for special behavior

**Pull request merged**